src/nbl/video/CSurfaceVulkan.cpp

```diff
@@ -143,5 +143,29 @@ namespace nbl::video
 			return nullptr;
 		}
 	}
+#elif defined(_NBL_PLATFORM_LINUX_)
+	
+	#include <xcb/xcb.h>
```

Comment:
for this include to work you need to remove the enclosing `namespace nbl::video {` and do 
```cpp
using namespace nbl;
using namespace nbl::video;
```
at the top instead